src/stompInstance.js

```diff
@@ -0,0 +1,7 @@
+import { Client } from '@stomp/stompjs';
+
+const stompInstance = new Client({
+  brokerURL: 'ws://localhost:8080/ws',
```

Comment:
dit moet in de dev misschien wel naar cherement wijzen

Reply:
whoops

Reply:
fixed